goteoapi/projects/swagger_specs/project_item.yml

```
@@ -200,6 +200,12 @@ definitions:
             amount:
                 type: number
                 description: Minimum amount required for a invest to apply this reward
+            units:
+                type: integer
+                description: Reward's item units
+            available_units:
```

```suggestion
            available-units:
```
Check the tests! are failing because of this.